jekyll/_cci2/env-vars.md

```diff
@@ -190,6 +190,65 @@ In every step, CircleCI uses `bash` to source `BASH_ENV`. This means that `BASH_
 The `$BASH_ENV` workaround only works with `bash`, and has not been confirmed to work with other shells.
 {: class="alert alert-info"}
 
+### Environment variable substitution
+{: #environment-variable-substitution }
+
+The CircleCI CLI offers a wrapper around the [`envsubst`](https://github.com/a8m/envsubst) tool, both locally and available in all jobs running on CircleCI. `envsubst` is a command-line utility used to replace environment variables in text strings.
+
+CLI command:
+
+```sh
+circleci env subst
+```
+
+#### Usage
+{: #env-subst-usage }
+
+Within your repository create a file such as `template.json`, with value replaced by environment variable strings
```

Comment:
Does the command require JSON specifically or does it accept certain other formats?

Reply:
Any text, it does not need to be a file of any type.